mmcls/models/losses/accuracy.py

```diff
@@ -19,8 +19,13 @@ def accuracy_numpy(pred, target, topk=1, thrs=0.):
     res = []
     maxk = max(topk)
     num = pred.shape[0]
-    pred_label = pred.argsort(axis=1)[:, -maxk:][:, ::-1]
-    pred_score = np.sort(pred, axis=1)[:, -maxk:][:, ::-1]
+    pred_label = pred.argpartition(-maxk, axis=1)[:, -maxk:]
+    pred_score = pred[np.indices((num, maxk))[0], pred_label]
```

Comment:
Maybe the `static_inds` can be reused here (line 23) if we don't need to support `pred.ndim` > 2